examples/heartbeat/src/main.py

```diff
@@ -0,0 +1,12 @@
+from kybra import heartbeat, query;
+
+initialized: bool = False
+
+@heartbeat
+def heartbeat_():
+    global initialized
+    initialized = True
+
+@query
+def getInitialized() -> bool:
```

Comment:
Can you make this camel case, and fix it in Azle as well?

Reply:
https://github.com/demergent-labs/azle/pull/770